alephium/caip10.md

```diff
@@ -0,0 +1,74 @@
+---
+namespace-identifier: alephium-caip10
+title: Alephium Namespace - Addresses
+author: Hongchao Liu (@h0ngcha0)
+discussions-to: https://github.com/ChainAgnostic/namespaces/pull/117
+status: Draft
+type: Standard
+created: 2024-06-19
+updated: 2024-06-19
+requires: ["CAIP-2", "CAIP-10"]
+---
+
+# CAIP-10
+
+*For context, see the [CAIP-10][] specification.*
+
+## Rationale
+
+An address on Alephium is a unique identifier that represents an
+account or a contract. All networks share the same address format.
+
+There are 4 different address types on Alephium:
+
+* 0x00 - Pay to public key hash (P2PKH)
+* 0x01 - Pay to multiple public key Hash (P2MPKH)
+* 0x02 - Pay to script hash (P2SH)
+* 0x03 - Pay to contract (P2C)
+
+For each address type, `content bytes` are defined as follows:
+
+- **P2PKH** - serialized public key hash
+- **P2MPKH** - serialized public key hashes and multisig threshold
+- **P2SH** - serialized script hash
+- **P2C** - serialized contract id
+
+Hash algorithm used is blake2b.
+
+Constructing an address:
+
+- **address** = `address type || content bytes`
+
+Each address on Alephium belongs to a group, which can be derived
+deterministically from the address.
```

Comment:
This is interesting (and reminds me of a similar property of Polkadot addresses) but for the CASA readership you need to spell this out a little more: link to (and summarize as concisely as possible) _how_ you derive group membership from the address (and whether an address is specific to mainnet/devnet or not, and whether that too can be derived diagnostically from an address).